oauth.go

```diff
@@ -350,6 +359,115 @@ func (x *OAuth) HttpHandlerOAuthTest(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// Provides support for username and password login to MSAAD
+// App to app authentication.
+// This is NOT recommended for normal user access but for trusted applications that have their own user credentials in MSAAD
+// AND is linked to the same tenant ID under which auth operates.
+
```

Comment:
Remove this empty line and add the method name to the beginning of the comment. This allows the IDE to pick up on the comment for this method.

Reply:
Cool. Will do.